src/Workspaces/Remote/ServiceHub/Host/RemoteWorkspace.cs

```diff
@@ -90,13 +83,12 @@ public AssetProvider CreateAssetProvider(Checksum solutionChecksum, SolutionAsse
             Func<Solution, ValueTask<T>> implementation,
             CancellationToken cancellationToken)
         {
-            return RunWithSolutionAsync(assetProvider, solutionChecksum, workspaceVersion: -1, updatePrimaryBranch: false, implementation, cancellationToken);
```

Comment:
📝 This instance of `workspaceVersion` was never used because `updatePrimaryBranch` is set to false and accesses of the version are gated by it.